src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -3660,6 +3660,23 @@ instruct vexpand(vReg dst, vReg src, vRegMask_V0 v0, vReg tmp) %{
   ins_pipe(pipe_slow);
 %}
 
+// ------------------------------ Vector signum --------------------------------
+
+// Vector Math.signum
+
+instruct vsignum_reg(vReg dst, vReg zero, vReg one, vRegMask_V0 v0) %{
+  match(Set dst (SignumVF dst (Binary zero one)));
+  match(Set dst (SignumVD dst (Binary zero one)));
+  effect(TEMP_DEF dst, TEMP v0);
+  format %{ "vsignum $dst, $dst\t" %}
+  ins_encode %{
+    BasicType bt = Matcher::vector_element_basic_type(this);
+    __ signum_fp_v(as_VectorRegister($dst$$reg), bt, Matcher::vector_length(this),
+                  as_VectorRegister($zero$$reg), as_VectorRegister($one$$reg));
```

Comment:
Nit: maybe leave one extra space here to align with parameters of the preceding line.

Reply:
> Ah, I would suggest remove them for test coverity reasons. We can add them back when needed.

That's a good point! the code is removed.